MediaBrowser.Controller/Providers/ImageRefreshOptions.cs

```diff
@@ -27,6 +27,11 @@ public ImageRefreshOptions(IDirectoryService directoryService)
 
         public bool IsAutomated { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the action is an identify.
+        /// </summary>
+        public bool IsIdentify { get; set; }
```

Comment:
`IsAutomated` just indicates that the refresh was triggered by the user (I think, it doesn't appear to do much from a brief scan). A normal manual metadata refresh, even with replace all metadata set, should not clear existing fields for which metadata is not found.

`IsIdentify` specifies that metadata refresh is the result of re-identifying the item, which should trigger a full wipe of downloaded metadata and images (because they were downloaded for the wrong id).

Reply:
IsIdentify is maybe a bit too specific though. `ClearOldMetadata` or whatever seems like a better fit to me